problems/prob2.rb

```diff
@@ -0,0 +1,15 @@
+#Given an integer array of size N, identify the largest 2 numbers in this array.
+#Do not sort the array.
+
+def largest_two_numbers(numbers)
+  if numbers.nil? || numbers.empty? || numbers.any? { |e| e == nil  }
+    return nil
+  elsif numbers.size == 1
+      return numbers
+  else
+    largest = numbers.max()
+    numbers.delete_at(numbers.index(largest))
```

Comment:
What issue is possible if the `numbers` array has a higher scope? `delete_at` mutates the array...

Reply:
I did it like that because in the problem description it didn't said that I was not allowed to destroy the array.

Reply:
Yes, but for getting 2 largest numbers, it is not justified enough, intuitively...